### PR TITLE
Localises more fields. 

### PR DESCRIPTION
**Pull request by smkvr98**
More fields from the UI localised. Deals with substep 1 of step 2 in issue #12. 
`pull out all strings from the UI just like 'giveFeedback'
naming conventions is 'controlName.shortId'`

### CODE REVIEW

File: src/pages/WordFrequency/WordFrequency.tsx [hunk 1]
```diff
@@ -31,7 +31,8 @@ export interface WordFrequencyParams {
 }
 
 export const WordFrequency = (props: RouteComponentProps<WordFrequencyParams>) => {
-  const [script] = H.useLocalStorageState<string>(PSC.Script.RO, 'currentScript')
+  const [scriptRaw] = H.useLocalStorageState<string>(PSC.Script.RO, 'currentScript')
+  const script = scriptRaw === 'xx' ? 'Latn' : scriptRaw
```
Comment (smkvr98) [inline src/pages/WordFrequency/WordFrequency.tsx:35]: should have been dealt with much before as part of substep 1  in step 1, my bad. (the drawer in the word frequency section behaved strange for locale xx.)

File: src/components/AppHeader/AppHeader.tsx [hunk 2]
```diff
@@ -50,7 +50,7 @@ export const AppHeader = ({ version, theme, toggleTheme, changeScript }: AppHead
 
   const [script, setScript] = H.useLocalStorageState<string>(PSC.Script.RO, 'currentScript')
 
-  const { t } = useTranslation()
+  const { t }: { t: any } = useTranslation()
```
Comment (smkvr98) [inline src/components/AppHeader/AppHeader.tsx:53]: work around for [this](https://github.com/mui-org/material-ui/issues/21218).

File: src/pages/WordFrequency/components/TipitakaHierarchyNodeDetails/TipitakaHierarchyNodeDetails.tsx [hunk 3]
```diff
@@ -76,7 +78,7 @@ export const TipitakaHierarchyNodeDetails = (props: TipitakaHierarchyNodeDetails
       <div className={classes.root}>
         <M.AppBar className={classes.appBar} position="static">
           <M.Tabs value={activeTab} onChange={handleChange}>
-            <M.Tab label="word frequency" {...a11yProps(0)} />
+            <M.Tab label={t`WordFreq.Word` + t`WordFreq.WFSpace` + t`WordFreq.Frequency`} {...a11yProps(0)} />
```
Comment (parthopdas) [inline src/pages/WordFrequency/components/TipitakaHierarchyNodeDetails/TipitakaHierarchyNodeDetails.tsx:81]: no plus business please. use string interpolation. did the linter pass even?

**Review by parthopdas — APPROVED** [review]
looks great.

only additional thing - it is worth checking if we can wrap the useTranslations hook with our own hook - that way we can wrap the t function with our version where it checks if current locale is xx and automatically adds ^ & $.

that way we can get rid of the xx.json file.

File: src/components/AppHeader/AppHeader.tsx (hunk 2 above)
Comment (parthopdas) [reply src/components/AppHeader/AppHeader.tsx:53]: what's wrong with a simple

const { t: any } = useTranslation()

**Comment by smkvr98** [comment]
I'll look into this.

**Pull request merged**